src/applications/resources-and-support/hooks/useGetSearchResults.js

```diff
@@ -16,6 +17,8 @@ export default function useGetSearchResults(articles, query, page) {
 
       const keywords = query
         .split(' ')
+        .filter(word => !!word)
+        .filter(word => !SEARCH_IGNORE_LIST.includes(word))
```

Comment:
What does `.filter(word => !!word)` filter out? I'm guessing a weird edge case?

Reply:
Yupp! So beforehand if you had 2 spaces it would consider one of them a word, so that filters out all `["", "", "help"]` in the array

Reply:
Oh interesting! Thanks for the insight!

Reply:
Wow, what an edge case!